barretenberg/cpp/src/barretenberg/sumcheck/instance/prover_instance.hpp

```diff
@@ -44,11 +44,14 @@ template <class Flavor> class ProverInstance_ {
     std::vector<FF> gate_challenges;
     FF target_sum;
 
-    ProverInstance_(Circuit& circuit, TraceStructure trace_structure = TraceStructure::NONE)
+    ProverInstance_(Circuit& circuit,
+                    TraceStructure trace_structure = TraceStructure::NONE,
+                    std::shared_ptr<typename Flavor::CommitmentKey> commitment_key = nullptr)
     {
         BB_OP_COUNT_TIME_NAME("ProverInstance(Circuit&)");
         circuit.add_gates_to_ensure_all_polys_are_non_zero();
         circuit.finalize_circuit();
+        info("finalized gate count: ", circuit.num_gates);
```

Comment:
should probably be deleted but I wish it was here so people would see what their gate counts are all the time.

Reply:
We are living with clientivc printing data, I'm sure we could live with this for now